dotcom-rendering/src/web/lib/useIsInView.ts

```diff
@@ -13,7 +16,10 @@ const useHasBeenSeen = (
 	// 200ms before the callback is executed
 	const intersectionFn: IntersectionObserverCallback = ([entry]) => {
 		if (entry.isIntersecting) {
-			setHasBeenSeen(true);
+			setIsInView(true);
+		}
+		if (options.repeat && !entry.isIntersecting) {
+			setIsInView(false);
 		}
```

Comment:
Not a suggestion, more of a question: is the following easier to read?

```ts
		if (entry.isIntersecting) {
			setIsInView(true);
		} else if (options.repeat) {
			setIsInView(false);
		}
```

Reply:
Much nicer! Done 👍